Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/Constants.java

```diff
@@ -0,0 +1,29 @@
+package org.firstinspires.ftc.team8923_PowerPlay;
+
+public class Constants {
+    //ratios
+    static final double TICKS_PER_REVOLUTION = 537.6; // Neverest orbital 20, 7 pulse per revolution
+    static final double GEAR_RATIO = 1.0;
+    static final double WHEEL_DIAMETER = 2.5; // inches
```

Comment:
Make sure to update the gear ratio and the wheel diameter when you know what they are. You can leave a TODO comment.

Reply:
Wheel diameter is 4 inches